### PR TITLE
Rewrite EFS tips section

### PR DESCRIPTION
**Pull request by bgdnlp**
As discussed on #299, I rewrote the tips section. Split mounting dirs and user access into two new tips and reworded some of it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -876,14 +876,17 @@ EFS
...
+	- Because performance is based on total size of stored files, having everything on one drive will increase performance for everyone. One application consuming credits faster than it can accumulate might be offset by another application that just stores files on EFS and rarely accesses them.
 	
 	Cons:
-	- Since credits are shared, if one application gets out of control it will affect the others.
-	- Security of the drive is compromised. All clients will have to have network access to the drive and any of the clients can mount another application's folder. Someone with root access on one client instance can easily read and write to any application's mount.
+	- Since credits are shared, if one application gets out of control consuming them, it will affect the others.
+	- A compromise is made with regards to [security](http://docs.aws.amazon.com/efs/latest/ug/security-considerations.html). All clients will have to have network access to the drive. Someone with root access on one client instance can mount any directory on the EFS and they have read-write access to all files on the drive, even if they don't have access to the aplications hosted on other clients.
```
Comment (ThanosBaskous) [inline README.md]: Spelling nit: "aplications" -> "applications"

File: README.md [hunk 2]
```diff
@@ -876,14 +876,17 @@ EFS
...
+-	⏱ One EFS can be used for multiple applications or services, but it should be considered carefully: 
 
 	Pros:
-	- Because performance is based on total size of stored files, having everything on one drive will increase performance for everyone and might offset one application consuming credits faster than it can accumulate by another application that just stores files on EFS and rarely accesses them.
+	- Because performance is based on total size of stored files, having everything on one drive will increase performance for everyone. One application consuming credits faster than it can accumulate might be offset by another application that just stores files on EFS and rarely accesses them.
 	
 	Cons:
-	- Since credits are shared, if one application gets out of control it will affect the others.
-	- Security of the drive is compromised. All clients will have to have network access to the drive and any of the clients can mount another application's folder. Someone with root access on one client instance can easily read and write to any application's mount.
+	- Since credits are shared, if one application gets out of control consuming them, it will affect the others.
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: "if one application over-consumes them,"

File: README.md [hunk 3]
```diff
@@ -876,14 +876,17 @@ EFS
 -	High availability is achieved by having [mount targets in different subnets / availability zones](http://docs.aws.amazon.com/efs/latest/ug/images/overview-flow.png).
 
 ### EFS Tips
--	⏱ Carefully consider using one EFS for multiple applications. Any directory on the EFS can be mounted, it doesn't have to be the root directory. One application could mount fs-12345678:/prog1, another fs-12345678:/prog2. [User and group level permissions](https://docs.aws.amazon.com/efs/latest/ug/accessing-fs-nfs-permissions.html) can be used to limit access to some extent.
+
+-   With EFS being based on NFSv4.1, any directory on the EFS can be mounted directly, it doesn't have to be the root directory. One application could mount *fs-12345678:/prog1*, another *fs-12345678:/prog2*. 
+-   [User and group level permissions](https://docs.aws.amazon.com/efs/latest/ug/accessing-fs-nfs-permissions.html) can be used to control access to certain directories on the EFS.
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: "...on an EFS file system."

"EFS file system" is how the docs in that page refer to an instance of EFS.

File: README.md [hunk 4]
```diff
@@ -876,14 +876,17 @@ EFS
...
+-	⏱ One EFS can be used for multiple applications or services, but it should be considered carefully:
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: "One EFS file system can be.."

File: README.md (hunk 3 above)
Comment (bgdnlp) [inline README.md]: Ha. Was wondering about that. Never occurred to me to check the official docs. What a novel approach :)

**Comment by jlevy** [comment]
👍 Looks good to me, and like all @ThanosBaskous 's suggestions are fixed, so merging. Great work!

**Pull request merged**